packages/auth/graphql/resolvers/User.js

```diff
@@ -32,7 +32,8 @@ module.exports = {
   },
   roles(user, args, { user: me }) {
     if (Roles.userIsMeOrInRoles(user, me, ['admin', 'supporter'])) {
-      return user.roles
+      const roles = user.roles ?? []
+      return roles.filter((role) => Roles.exposableRoles.includes(role))
```

Comment:
Isn't that unnecessary because we already do it in [transformUser](https://github.com/orbiting/backends/blob/master/packages/auth/lib/transformUser.ts#L36)?
```suggestion
      return user.roles.filter((role) => Roles.exposableRoles.includes(role))
```

Since the schema mandates `[String!]!` we would know if it often returned null. And this should always be getting transformed users.

Reply:
That is unnecessary. As far as I can tell all `User` wander through `transformUser`.